tasks/eth/008-fp-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,62 @@
+# VALIDATION
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
+
+### `0x229047fed2591dbec1ef1118d64f7af3db9eb290` (`SystemConfigProxy`)
+
+Links:
+- [Etherscan](https://etherscan.io/address/0x229047fed2591dbec1eF1118d64F7aF3dB9EB290)
+- [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/94149a2651f0aadb982802c8909d60ecae67e050/superchain/extra/addresses/mainnet/op.json#L10)
+
+State Changes:
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x000000000000000000000000ba2492e52f45651b60b8b38d4ea5e2390c64ffb1` <br/>
+  **After:** `0x000000000000000000000000f56d96b2535b932656d3c04ebf51babff241d886` <br/>
+  **Meaning:** This upgrades the SystemConfig implementation. Verify that the new `SystemConfig` implementation is stoored at the eip1967 proxy implementation slot.
+
+- **Key**: `0x52322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa906`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000000000000`
+  **After**: `0x000000000000000000000000e5965ab5962edc7477c8520243a95517cd252fa9`
+  **Meaning**: Sets the [DisputeGameFactory slot](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.4.0-rc.4/packages/contracts-bedrock/src/L1/SystemConfig.sol#L76). Verify that the `Key` matches the expected slot.
```

Comment:
Suggest adding the following to validate the Key:

---

You can verify the correctness of the storage slots with `chisel`.

Just start it up and enter the slot definitions as found in the contract source code.
```
➜ bytes32(uint256(keccak256("systemconfig.disputegamefactory")) - 1)
Type: bytes32
└ Data: 0x4aaefe95bd84fd3f32700cf3b7566bc944b73138e41958b5785826df2aecace0
```